AeroGearOAuth2/WebViewController.swift

```diff
@@ -0,0 +1,43 @@
+/*
+* JBoss, Home of Professional Open Source.
+* Copyright Red Hat, Inc., and individual contributors
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import Foundation
+
+import UIKit
+
+class WebViewController: UIViewController, UIWebViewDelegate {
+    var targetURL : NSURL = NSURL()
+    var webView : UIWebView = UIWebView()
+      
+    override internal func viewDidLoad() {
+        super.viewDidLoad()
+        webView.frame = UIScreen.mainScreen().applicationFrame
+        webView.scalesPageToFit = true
```

Comment:
notice if the controller is switched in landscape the web view doesn't correctly "scale-to-fit"